pkgs/development/python-modules/tensorstore/default.nix

```diff
@@ -0,0 +1,23 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  numpy,
+  setuptools-scm,
+}:
+buildPythonPackage rec {
+  pname = "tensorstore";
+  version = "0.1.22";
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-qQqytAtjXtThWh+a+dNLeQHlxYUGxdUOgK69aFDQtwA=";
+  };
+  buildInputs = [setuptools-scm];
+  propagatedBuildInputs = [numpy];
+  meta = with lib; {
```

Comment:
Also check if you could ass `pythonImportsCheck`. Explicitly specifying `checkInputs = [ pytestCheckHook ];` is also recommended. If the tests are missing in the pypi release, you might have to fetch the sources from the GitHub repo

Reply:
I think I did this correctly, but please let me know.